_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title:  "Searchable snapshots are generally available in OpenSearch 2.7"
+authors:
+  - satnandi
+date:   2023-04-27
+categories:
+  - technical-post
+meta_keywords: searchable snapshots, search data segment in snapshots, searchable snapshot index, OpenSearch 2.7
```

Comment:
```suggestion
meta_keywords: searchable snapshots, search data in snapshots, searchable snapshot index, OpenSearch 2.7
```

Reply:
@anasalkouz - Please change the meta back to this as the words are drawn from the text and how people search - not preference based. 

meta_keywords: searchable snapshots, **search data segment in snapshots**, searchable snapshot index, OpenSearch 2.7